package.json

```diff
@@ -38,7 +38,7 @@
     "@formatjs/intl-locale": "^2.4.21",
     "@formatjs/intl-numberformat": "^6.2.5",
     "@formatjs/intl-pluralrules": "^4.0.13",
-    "@onfido/react-native-sdk": "^1.3.3",
+    "@onfido/react-native-sdk": "^2.2.0",
```

Comment:
I think there might be some pod changes for this bump.

Also maybe we should do some QA? At least check to see that the Onfido flows are working on native.

Reply:
Thanks, good points. I was so relieved to have Android succeed I forgot about iOS. Will add notes about onfido testing too.